envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/permissions/TlsClientCertRequiredTest.kt

```diff
@@ -30,28 +30,26 @@ class TlsClientCertRequiredTest {
 
         @JvmField
         @RegisterExtension
-        val envoy = EnvoyExtension(envoyControl, config = Echo1EnvoyAuthConfig)
+        val service = EchoServiceExtension()
```

Comment:
Why is TLS test changing when the PR is bout path regex? Just refactoring?

Reply:
No, this test was simply broken and didn't work at all. I discovered it when I was debugging the new test which was not working for the same reason.